packages/openapi-fetch/src/index.js

```diff
@@ -231,6 +207,19 @@ export default function createClient(clientOptions) {
       }
     },
   };
+
+  const handler = {
+    get: (coreClient, property) => {
+      if (property in coreClient) {
+        return coreClient[property];
+      }
+
+      // Assume the property is an URL.
+      return Object.fromEntries(methods.map((method) => [method, (init) => coreFetch(property, { ...init, method })]));
```

Comment:
Performance regression: taking previously-static properties, and iterating over them _on every fetch call_ will mean a big performance hit when it’s all added up. Every time we convert a static object with known properties into another loop means the overall runtime extends by a little bit each time. Prefer “[WET](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#WET)” code here when possible (and again, this isn’t the norm on _all_ codebases; just library code)!